docs/pages/en-us/Getting-Started.md

```diff
@@ -254,118 +254,17 @@ Regardless of whether you choose to use a generated client-side library, or to u
-	
-	    BootstrapConfiguration bootstrap = new SystemBootstrapConfiguration();
-	    Cipher cipher = ModuleLoader.getService(CipherProvider.class, NullCipherProvider.class).getCipher(bootstrap);
-	    ServicePath path = bootstrap.getServicePath();
-	    Codec codec = ModuleLoader.getService(Codec.class);
-	    return ModuleLoader.getService(ConfigurationSourceProvider.class).getConfigurationSource(bootstrap, cipher, path, codec);
-	}
-
-To compile the above sample code, add the following dependencies to your POM:
-
-	<dependency>
-	    <groupId>com.github.cafapi</groupId>
-	    <artifactId>caf-api</artifactId>
-	    <version>1.6.0-176</version>
-	</dependency>
-	<dependency>
-	    <groupId>com.github.cafapi.cipher</groupId>
-	    <artifactId>cipher-null</artifactId>
-	    <version>1.6.0-176</version>
-	</dependency>
-	<dependency>
-	    <groupId>com.github.cafapi.config</groupId>
-	    <artifactId>config-system</artifactId>
-	    <version>1.6.0-176</version>
-	</dependency>
-	<dependency>
-	    <groupId>com.github.cafapi.util</groupId>
-	    <artifactId>util-moduleloader</artifactId>
-	    <version>1.6.0-176</version>
-	</dependency>
-	<dependency>
-	    <groupId>com.github.cafapi.util</groupId>
-	    <artifactId>util-naming</artifactId>
-	    <version>1.6.0-176</version>
-	</dependency>
-
-To use JSON-encoded files for your configuration, add the following additional dependencies to your POM:
-
-	<!-- Runtime-only Dependencies -->
-	<dependency>
-	    <groupId>com.github.cafapi.config</groupId>
-	    <artifactId>config-file</artifactId>
-	    <version>1.6.0-176</version>
-	    <scope>runtime</scope>
-	</dependency>
-	<dependency>
-	    <groupId>com.github.cafapi.codec</groupId>
-	    <artifactId>codec-json</artifactId>
-	    <version>1.6.0-176</version>
-	    <scope>runtime</scope>
-	</dependency>
-	<dependency>
-	    <groupId>io.dropwizard</groupId>
-	    <artifactId>dropwizard-core</artifactId>
-	    <version>0.8.4</version>
-	    <scope>runtime</scope>
-	</dependency>
-
-#### Configuration Required for the AuditConnection
-
-In the `ConfigurationSource` above, we used JSON-encoded files with the following parameters:
-
-- `CAF_CONFIG_PATH: /etc/sampleapp/config`
-- `CAF_APPNAME: sampleappgroup/sampleapp`
-
-Given this configuration, you would configure Auditing by creating a file named `cfg_sampleappgroup_sampleapp_ElasticAuditConfiguration` in the `/etc/sampleapp/config/` directory. The contents of this file should be similar to the following:
-
-	{
-	    "hostAndPortValues": "<Elasticsearch_Cluster_Node1>:<ES_Port_Node1>,<Elasticsearch_Cluster_Node2>:<ES_Port_Node2>,<Elasticsearch_Cluster_Node3>:<ES_Port_Node3>",
-	    "clusterName": "elasticsearch-cluster",
-	    "numberOfShards": "5",
-	    "numberOfReplicas": "1"
-	}
+	- `CAF_ELASTIC_NUMBER_OF_REPLICAS` the number of replica shards (copies) that each primary shard should have. Defaults to 1.  
+Or the following environment variable when using with the webservice audit implementation:  
```

Comment:
Indenting doesn't look right.  WS config should not be subordinate to ES config.